external/netstandard/netstandard.depproj

```diff
@@ -60,7 +60,8 @@
       <Version>4.5.0</Version>
     </PackageReference>
     <PackageReference Include="System.Security.Cryptography.Pkcs">
-      <Version>4.5.0</Version>
+      <!-- netstandard2.0 ref had extra (MissingMethodException) property setters which were removed in 4.5.1 -->
```

Comment:
I get that 4.5.1 is sufficient here, but should we just upgrade this to 4.5.2? Should we do the same for Cng and OpenSsl?

@ericstj I believe that we should grep for all the places where we use package versions outside of the harvesting projects, and move all of those to properties in Versions.props so the whole repo is updated whenever we re-baseline, so that we don't hit issues like the one here.

Reply:
Yes, it should reference latest servicing release where possible.